src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Buldozer stopped working");

```

Comment:
The message 'Buldozer stopped working' should also be more informative about what the machine was doing before it stopped.